### DISCUSSION TITLE
Sync specific albums only

### DISCUSSION THREAD

**Discussion by halbtuerke** [steilerDev/icloud-photos-sync#328] [Q&A] [answered]
Hi,

is it possible to only specific albums?

Thanks

**Comment by steilerDev** [comment, marked as answer]
@halbtuerke nope - all or nothing at the moment.

It's treating assets independently from albums - so it will first fetch all assets in your personal and shared library and then link the assets into a folder structure. 

Excluding specific assets based on them being in a folder is currently not in scope. 

You should however be able to extract them easily by copying them out of the folder after syncing. Alternatively you could `archive` them (which optionally deletes the folder remotely) and move them out of the folders afterwards.

**Reply by halbtuerke** [reply]
Thank you for the clarification.
